File: 2) JAVA/soma.java
import java.util.Scanner;

public class Main {
    public static void main( String[] args ) {
        Scanner scanner = new Scanner( System.in );

        System.out.println( "Digite o primeiro valor: " );
        int numero1 = scanner.nextInt( );

        System.out.println( "Digite o segundo valor: " );
        int numero2 = scanner.nextInt( );

        int soma = numero1 + numero2;

        System.out.println( "Soma: " + soma );
        
        scanner.close( );
    }
}

File: 4) C#/soma.cs
using System;

namespace Soma
{
	class Program
	{
		static void Main( string[ ] args )
		{
			int num1, num2, soma;
			Console.WriteLine( "Digite o primeiro valor: ");
			num1 = Convert.ToInt32(Console.ReadLine( ));
			Console.WriteLine( "Digite o segundo valor: ");
			num2 = Convert.ToInt32(Console.ReadLine( ));
			
			soma = num1 + num2;
			
			Console.WriteLine( "Soma: " + soma );
		}
	}
}	